### DISCUSSION TITLE
i3 not loading after login

### DISCUSSION THREAD

**Discussion by j0ker70** [i3/i3#4839] [Q&A]
I am using i3 on Ubuntu 20.04. Recently I installed picom and replaced compton with picom in the i3 config file.After the restarted i3(ctrl + shift + r). After that my  laptop hanged. I restarted my laptop. I logged in. But i3 is not loading. I am getting a blank screen. I also tried Ubuntu as my DE but that also does the same thing. Only thing I have is a tty terminal. But I don't how to fix this.

**Comment by orestisfl** [comment]
Try removing picom from the config file. If it works, execute picom from a terminal, changing the settings until it works right.

**Reply by j0ker70** [reply]
I did it. But still not working. Its worse now. It freezes before the login screen comes. I can't use any tty now.

**Reply by orestisfl** [reply]
Ok, start by restoring the original i3 config and sharing your xinitrc

**Reply by j0ker70** [reply]
Don't have tty terminal now. My laptop keeps freezing before the login.

**Comment by Sal-Kal** [comment]
Any update here? Have been facing the same issue. 

Also I have been facing this issue with only Xorg based desktop environments. When I log in to Ubuntu Wayland or try to launch Sway, it seems to work fine.

Also when I am greeted with the black screen after logging in, it doesn't stay black forever, i3wm starts after somewhere around 5 mins.
